lib/rack/jwt/auth.rb

```diff
@@ -64,25 +64,30 @@ def initialize(app, opts = {})
       end
 
       def call(env)
-        return @app.call(env) if path_matches_excluded_path?(env)
+        auth_is_required = !path_matches_excluded_path?(env)
 
-        if auth_cookie_enabled? && missing_auth_cookie?(env) && missing_auth_header?(env)
-          return return_error('Missing token cookie and Authorization header')
-        end
-        if auth_cookie_enabled? && empty_auth_cookie?(env)
-          return return_error('Empty token cookie')
-        end
-        if auth_cookie_disabled? && missing_auth_header?(env)
-          return return_error('Missing Authorization header')
-        end
-        if auth_cookie_disabled? && invalid_auth_header?(env)
-          return return_error('Invalid Authorization header format')
+        if auth_is_required
```

Comment:
```suggestion
        if auth_is_required?
```

Reply:
this is a local variable, i can't use a trailing `?` in the name.  i can turn it into a function, but I will have to do the exclude list lookup twice.

Reply:
Ah, I hadn't notced it was a variable. Fair enough, then.